### PR TITLE
fix: Fix tour step type

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/39307

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Fix tourStep set `type=‘primary’` not work|
| 🇨🇳 Chinese |    修复 TourStep 设置 type=‘primary’ 无效的问题|

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
> 修复tour step type

看不太懂。

**Comment by afc163** [comment]
来个 type.test.tsx 用例？

**Comment by MadCcc** [comment]
要补一下 step 单独传 type 的用例

**Comment by heiyu4585** [comment]
> 来个 type.test.tsx 用例？
sorry,两位,我先占位.先过rc-tour的

**Comment by MadCcc** [comment]
semver 要改了

**Comment by MadCcc** [comment]
snapshot 整体应该不变吧

**Comment by MadCcc** [comment]
Test 还是挂的

**Comment by heiyu4585** [comment]
> Test 还是挂的

恩，在等豆酱大佬的PureRender

**Comment by heiyu4585** [comment]
> Test 还是挂的

收到，豆酱大佬跟进PureRender

File: package.json [hunk 1]
```diff
@@ -115,7 +115,8 @@
     "@ant-design/react-slick": "~0.29.1",
     "@babel/runtime": "^7.18.3",
     "@ctrl/tinycolor": "^3.4.0",
-    "@rc-component/tour": "~1.0.1-2",
+    "@rc-component/tour": "~1.1.0",
+    "antd-token-previewer": "^1.1.0-21",
```
Comment (li-jia-nan) [inline package.json:119]: 这个貌似是多余的，devDependencies 里面已经有了

**Comment by li-jia-nan** [comment]
\rebase

**Review by MadCcc — APPROVED** [review]

File: components/tour/style/index.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ const genBaseStyle: GenerateStyle<TourToken> = (token) => {
           },
         },
 
-        // ============================= primary type ===========================
+        // =============================  primary type  ===========================
```
Comment (afc163) [inline components/tour/style/index.tsx:155]: 为啥要改这个

Comment (li-jia-nan) [reply components/tour/style/index.tsx:155]: 强推一下重跑CI

Comment (yoyo837) [reply components/tour/style/index.tsx:155]: 触发 https://github.com/ant-design/ant-design/pull/39382/commits/036c8aac55cdf2ef8166d22b9316b38f7f122aed , 没推空提交, 改了注释来达到目的,

**Comment by afc163** [comment]
changelog 按用户视角改改

**Comment by heiyu4585** [comment]
> changelog 按用户视角改改

done

**Pull request merged**